src/content-scripts/components/RepoQuery/RepoQuery.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { RepoQuery } from "../../../repo-query/pages/main";
+import { createHtmlElement } from "../../../utils/createHtmlElement";
+
+export const RepoQueryRoot = (ownerName: string, repoName: string) => {
+    const repoQueryRoot = createHtmlElement("div", {
+        id: "repo-query-root",
+        innerHTML: `
+            <div>
+            </div>
+        `,
+    });
```

Comment:
Maybe I'm missing something, bit does the repo query root (a `<div />`) need a div in itself? Shouldn't the main element be enough for the root, i.e. `<div id="repo-query-root"></div>`?

```suggestion
    const repoQueryRoot = createHtmlElement("div", {
        id: "repo-query-root",
    });
```